README.md

```diff
@@ -154,7 +154,7 @@ and tools.
 
 To build Filament, you must first install the following tools:
 
-- CMake 3.4 (or more recent)
+- CMake 3.13 (or more recent)
```

Comment:
Maybe we could relax this requirement for users not building Android?

Reply:
I thought about it but wanted to keep things simple.

Reply:
I think the Android build should check for this requirement explicitly and fail with a useful message. The build errors I got were very opaque with cmake 3.10 and I took a circuitous route to get here.

Reply:
build.sh now makes the check.